lib/from_erlang.ml

```diff
@@ -281,9 +280,19 @@ and function_of_clauses clauses =
       let tuple_pattern = PatTuple ps in
       (((tuple_pattern, Constant (line, Atom ("true"))), expr_of_erlang_expr' body), arity)
     ) |> List.unzip in
+     let line_number_of_clause = (function
+     | ((PatTuple _patterns, _), term) -> Ast.line_number_of_t term
+     | ((PatConstant _, _), term) -> Ast.line_number_of_t term
+     | ((PatCons (_, _), _), term) -> Ast.line_number_of_t term
+     | ((PatVar _, _), term) -> Ast.line_number_of_t term
+     | ((PatNil, _), term) -> Ast.line_number_of_t term
+     ) in
      let make_fresh_variables length = fill (fun () -> Variable.create()) length |> List.rev in
      let make_case cs fresh_variables =
-       let fresh_tuple = Tuple (fresh_variables |> List.map ~f:(fun v -> Var (-1, v))) in
+       let fresh_tuple = Tuple (
+         line_number_of_clause (List.hd_exn cs),
+         fresh_variables |> List.map ~f:(fun v -> Var (-1, v))
```

Comment:
I think it is convenient for debugging to specify line number as possible.
So, how about this?

```ocaml
  let l = line_number_of_clause (List.hd_exn cs) in
  let fresh_tuple = Tuple (l, fresh_variables |> List.map ~f:(fun v -> Var (l, v))) in
```

Reply:
OK.  I'll do it.